src/java.desktop/share/classes/sun/java2d/pipe/LoopPipe.java

```diff
@@ -261,17 +261,25 @@ public static ShapeSpanIterator getStrokeSpans(SunGraphics2D sg2d,
         ShapeSpanIterator sr = new ShapeSpanIterator(false);
 
         try {
-            sr.setOutputArea(sg2d.getCompClip());
+            final Region clip = sg2d.getCompClip();
+            sr.setOutputArea(clip);
             sr.setRule(PathIterator.WIND_NON_ZERO);
 
             BasicStroke bs = (BasicStroke) sg2d.stroke;
             boolean thin = (sg2d.strokeState <= SunGraphics2D.STROKE_THINDASHED);
             boolean normalize =
                 (sg2d.strokeHint != SunHints.INTVAL_STROKE_PURE);
 
+// No clipping (pre-jdk17)
```

Comment:
This code is not a part of the marlin, so exists only in the jdk17. Do we need these commented lines here?

Reply:
Will remove the comment